ansible_collections/arista/avd/roles/eos_designs/doc/fabric-variables.md

```diff
@@ -46,7 +49,7 @@ shutdown_interfaces_towards_undeployed_peers: < true | false | default -> false
 
 # Underlay routing protocol | Required.
 underlay_routing_protocol: < EBGP | OSPF | ISIS | ISIS-SR | ISIS-LDP | ISIS-SR-LDP | OSPF-LDP | none | default for l3ls-evpn -> EBGP >
-overlay_routing_protocol: < EBGP | IBGP | none | default for l3ls-evpn -> EBGP >
+overlay_routing_protocol: < EBGP | IBGP | none | her | default for l3ls-evpn -> EBGP >
```

Comment:
same remark as above for HER instead of her

Reply:
done